packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.controller.js

```diff
@@ -15,6 +23,22 @@ export default class {
     this.loader = {
       updatingCapabilities: false,
     };
+    this.$timeout(() => this.createProgressAlert());
```

Comment:
Is there a reason to trigger it after a delay? In the constructor also?

Reply:
This method creates the alerts required for the page. If done without the `$timeout`, the messages are created before the dom is fully rendered and it causes the messages to not be displayed

Reply:
Maybe handling this in a `$postLink` hook would work? (PR is opened since more than one month, so no matter if you let this as is)